libpod/runtime_img.go

```diff
@@ -214,9 +312,16 @@ func (r *Runtime) PushImage(source string, destination string, options CopyOptio
 		signaturePolicyPath = options.SignaturePolicyPath
 	}
 
-	policyContext, err := common.GetPolicyContext(signaturePolicyPath)
+	sc := common.GetSystemContext(signaturePolicyPath, options.AuthFile)
```

Comment:
This `sc` is only used for `signature.DefaultPolicy`, which does not use `AuthFile`.  `AuthFile` needs to be set in `copy.Options.{Source,Destination}Ctx`, and that is not happening AFAICS.

Reply:
Another for @mtrmac

Reply:
this was fixed.